refs/schemas/priv/expected-behavior.md

```diff
@@ -0,0 +1,655 @@
+CONTACT.EMAIL x * x MARKETING (LEGITIMATE-INTEREST)
+CONTACT.EMAIL x * x SERVICES (CONTRACT)
+CONTACT.ADDRESS x * x SERVICES (CONTRACT)
+```
+
+The Data Subject realizes that they are still receiving marketing e-mail and they formulate a Privacy Request with one `OBJECT` demand with `CONTACT.EMAIL` being defined as its Privacy Scope. Yet, this Demand only affects the [Privacy Scope Triples](#privacy-scope-triples) held under `LEGITIMATE-INTEREST`. The System can still, continue to keep the `CONTACT.EMAIL` and `CONTACT.ADDRESS` for the same of the ongoing contractual relationship with the user.
+
+```
+Eligible Privacy Scope:
+CONTACT.EMAIL x * x SERVICES (CONTRACT)
+CONTACT.ADDRESS x * x SERVICES (CONTRACT)
+```
+However, the remaining purpose (`SERVICES`) now only allows the System to send e-mail related to the services being provided, and no longer supports sending marketing e-mail to this Data Subject.
+
+### Calculating Permissions
+
+Since the [Privacy Compiler](../high-level-architecture#data-rights-compiler) maintains the Eligible Privacy Scope at any time, it can (given a Data Capture ID, or a Data Capture Fragment ID) answer whether at that time, a particular data processing operation is permitted.
+
+For example, the System can inquire whether a particular data fragment, corresponding to `CONTACT.EMAIL` data category, can be used to invoice the user (processing = `USING`; purpose = `SERVICES`), or whether it can be shared with some other partner system for user profiling (processing = `SHARING`; purpose = `AUTOMATED-INFERENCE`).
+
+
+## Resolving requests
+
+The [Privacy Compiler](../high-level-architecture#data-rights-compiler) evaluates Privacy Request Demands and issues recommendations to the System. The Systems can then be configured to respond to Privacy Requests automatically, blindly following those recommendations (whenever possible), or to await human input just in case. Cf. [different automation scenarios](./scenarios.md#automation)
+
+Privacy Requests MAY need to be processed in the context of different [authentication scenarios](./scenarios.md#authentication) and different [response scenarios](./scenarios.md#response).
+
+Here is how the Privacy Request Response recommendations are calculated:
+
+When no Data Subject ID is provided in the Privacy Request:
+- `TRANSPARENCY` Demands: recommend status = `GRANTED`. Provide requested information: Data Categories, Processing Categories, Purposes, and Legal Bases from Intended Privacy Scope, as well as other information from general settings as defined under [Implications for Systems](./RFC-PRIV.md#design-implications-for-systems-implementing-PRIV) and under [Configuration and Prerequisites](#configuration-and-prerequisites).
+- `OTHER` Demands: recommend human review and status = `UNDER-REVIEW`
+- For any other action : recommend status = `DENIED`, motive=`IDENTITY-UNCONFIRMED`
+
+When Data Subject ID is provided, Data Subject is authenticated but is unknown by the System:
```

Comment:
Thinking about how can a Data Subject be authenticated by the System but not known to it? Can you give an example.

I imagine it is possible to authenticate a Data Subject via document id (e.g., passport) inspection by Data Consumers, is this what you have in mind? Looking at the diagrams [here](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/scenarios.md#a-priori-authentication) my understanding is that authentication here means automatic authentication without user intervention.

Reply:
>Thinking about how can a Data Subject be authenticated by the System but not known to it? Can you give an example.

This can happen under many different circumstances, examples of which include:
- In [Single-Point authentication](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/scenarios.md#signle-point-authentication-for-corresponding-systems), System A authenticates the User to be indeed the Data Subject. It tranferes the Privacy Request to System B (which trusts System A's claim that User=Data Subject), however System B does not have any data on the Data Subject in question. 
    - Maybe System A transfered data about this Data Subject to System B, but then Data Subject directly made a privacy Request to DELETE data directly at System B. When the Data Subject goes to System A with a request, System A might still be obliged to transfer the request to System B, where the Data Subject is no longer known.
- In [A Priori Authentication](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/scenarios.md#a-priori-authentication) scenarios, regardless of authentication scenario being [Signle-point]https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/scenarios.md#signle-point-authentication-for-corresponding-systems) or [Independent](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/scenarios.md#independent-authentication-for-corresponding-systems), the Privacy Request Capture Interface can authenticate the user as indeed being the Data Subject (e.g., bu using magic link, or by using human validation service for pictures of ID documents), yet there is no data about this user in the System at all.

> I imagine it is possible to authenticate a Data Subject via document id (e.g., passport) inspection by Data Consumers, is this what you have in mind? Looking at the diagrams [here](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/scenarios.md#a-priori-authentication) my understanding is that authentication here means automatic authentication without user intervention.

The diagrams make aboslutely no reference on how the user is autehnticated. They amke abstraction over the authentication method, and are made to suggest no authentication method in particular. These scenarios MUST be supported whatever the authentication method use, including:
- magic links
- oauth
- user-account
- 3rd party ID-verification service
- inhous Data Consumer/DPO-based ID-verification
- etc.